devicetypes/smartthings/zigbee-stateless-curtain-window-shade.src/zigbee-stateless-curtain-window-shade.groovy

```diff
@@ -0,0 +1,101 @@
+/**
+ *
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+import physicalgraph.zigbee.zcl.DataType
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+metadata {
+    definition(name: "Zigbee Stateless Curtain Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-curtaion") {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Switch Level"
+        capability "Stateless Curtain Power Button"
```

Comment:
any reason why you're using this capability and not `Window Shade`? The `pause` command was just added.

Reply:
For the capability  `window shade`, it can't `pause` when the status is `opening` or `closing`. But we need to control it pause when the window shade is during `opening` or `closing`. the capability `Stateless Curtain Power Button` is better for this situation.

Reply:
Huh? @BYDengS Where did you hear that from?

Reply:
@dkirker , I have used the capability `window shade` before, and found that there is only one button in the screen, and there is some case that one button can't be satisfied. For example, when the window shade is about "50%" status, I have two choices （open or close）, but the only one button can't do that.

Reply:
@BYDengS The capability was *just* updated, so I don't believe the metadata has been updated to include pause. I believe the pause command still needs to be mapped.

Reply:
I used `sendEvent(name: "availableCurtainPowerButtons", value: ["open", "pause", "close"])` to add three button for this capability, and in the ST app screen, there are three button shows, and the button event is mapping for setButton() function, and I checked the three button, it works well.

Reply:
@dkirker And more information is in this ticket, https://smartthings.atlassian.net/browse/ICP-9463?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

Reply:
and also this one, https://smartthings.atlassian.net/browse/CPBLTS-1318

Reply:
@BYDengS Does the device actually report back how open it is? (I see you are checking level in parse() )?

And I am familiar with CPBLTS-1318. I created the definition. I checked and see that the metadata generator has the pause() command, and it appears that the mapping task is "Waiting for QA".

See CPBLTS-1319 for the final definition.

As for only one button, I agree. We should bring that up with UX. Any chance you could file a bug and assign it to @ManHyun-Hwang ? I know there should be a slider to set the level as you have defined Switch Level. In theory, if you wanted the shade more open you would drag the slider.

Reply:
@dkirker  The device just reported the level percentage. And the task for this curtain is urgent, can we use the capability `Stateless Curtain Power Button`? if change the capability to `window shade`, it will spend a lot of time.

Reply:
My understanding is that most of the work is done.

Reply:
Let me check the status of the mapping when it is not 2am and see what state the update to `Window Shade` is in. Since we just added a command, it might be working.

Reply:
Ok, thank you :)